pkg/config/config.go

```diff
@@ -155,7 +155,7 @@ func readConfigOrSecret(secretName, configPath string, v *viper.Viper) error {
 		}
 	}
 
-	for k, v := range viper.AllSettings() {
+	for k, v := range v.AllSettings() {
```

Comment:
Can we either change the `v` in the loop or `v` for viper just so it reads a little better?

Reply:
Oh duh right, no variable shadowing. Too bad we never added a linter :)